### DISCUSSION TITLE
Will firefox-ios describe privacy manifest files (PrivacyInfo.xcprivacy)

### DISCUSSION THREAD

**Discussion by nguyenninhit** [mozilla-mobile/firefox-ios#17162] [General]
I can not find privacy manifest files in your source codes. Will we support it in the future or have I missed something?

`From Spring 2024, apps that don’t describe their use of required reason API in their privacy manifest file won’t be accepted by App Store Connect.`
https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api

Many thanks.

**Comment by lmarceau** [comment]
Hello @nguyenninhit, this still need to be added into our code to follow the App Store requirement. We're aware of this and will add it 🙏 Thank you!
